src/java/org/apache/nutch/util/ProtocolStatusStatistics.java

```diff
@@ -122,8 +122,17 @@ public int run(String[] args) throws Exception {
     job.setNumReduceTasks(numOfReducers);
 
     try {
-      job.waitForCompletion(true);
-    } catch (Exception e) {
+      boolean success = job.waitForCompletion(true);
+      if(!success){
```

Comment:
Please use consistent and uniform formatting.

Reply:
@sebastian-nagel I did not understand, the formatting looks good to me. Can you please elaborate on what I am missing here? Thanks.

Reply:
There is a missing space before opening brackets/braces (also other occurrences). The formatting should follow the style defined by the Eclipse Code Formatter rules (https://github.com/apache/nutch/blob/master/eclipse-codeformat.xml). The easiest way is to import the rules into Eclipse (and other IDEs), in doubt, Eclipse allows to format the code from the command-line.

Reply:
Apologies for such a silly query 😅 . I use vim editor and everything lgtm, but, after importing the project into eclipse I could see the differences in the format. Thanks.